src/HDF5.jl

```diff
@@ -1127,7 +1160,7 @@ function a_read(parent::Union{File,Group,Dataset,Datatype}, name::AbstractString
     ret
 end
 
-# Datafile.jl defines generic read for multiple datasets, so we cannot simply add properties here.
+# H5DataStore defines generic read for multiple datasets, so we cannot simply add properties here.
```

Comment:
This comment isn't necessary anymore, right? With keywords rather than key-value argument pairs, there's not conflict.

Reply:
That's right. 

I think  these two function need to pass  the `pv`  keyword list down to the `read` function as well.
https://github.com/JuliaIO/HDF5.jl/pull/724/files#diff-3df1923ba1a5ee433e521ee7d3f75d066f7674b297db94ecda3ad89f7fc3e9a2R65 and https://github.com/JuliaIO/HDF5.jl/pull/724/files#diff-3df1923ba1a5ee433e521ee7d3f75d066f7674b297db94ecda3ad89f7fc3e9a2R70

Reply:
Another minor thing I think we should change is that the method here should probably be private and use an underscore prefix
https://github.com/JuliaIO/HDF5.jl/blob/00cf08352b0640b5b2a25ac10845c22fb167231c/src/HDF5.jl#L440

Reply:
I'd say leave the potential change to `read(::H5DataStore, names...; pv...)` to a different PR — one could argue that as generic convenience functions, the interface doesn't need to accept any property values, and if you require property uses you should use the more specific functions that operate on only a single dataset at a time.